build-info-extractor-docker/src/main/java/org/jfrog/build/extractor/docker/types/DockerImage.java

```diff
@@ -132,11 +131,9 @@ private void setDependenciesAndArtifacts(Module buildInfoModule, Properties prop
         for (int i = 0; i < dependencyLayerNum; i++) {
             String digest = it.next();
             DockerLayer layer = layers.getByDigest(digest);
-            HttpResponse httpResponse = propertyChangeClient.executeUpdateFileProperty(layer.getFullPath(), artifactsPropsStr);
-            validateResponse(httpResponse);
-            Dependency dependency = new DependencyBuilder().id(layer.getFileName()).sha1(layer.getSha1()).addProperties(properties).build();
+            Dependency dependency = new DependencyBuilder().id(layer.getFileName()).sha1(layer.getSha1()).build();
             dependencies.add(dependency);
-            Artifact artifact = new ArtifactBuilder(layer.getFileName()).sha1(layer.getSha1()).addProperties(properties).build();
```

Comment:
Can't we set custom properties on pushed docker images?

Reply:
Yes, but here we deal with the buildinfo.json artifacts & dependencies.

Reply:
Where do you set the properties? I couldn't find

Reply:
https://github.com/jfrog/build-info/pull/397/files#diff-f2b37fdb29ea988c82ea3a71485db026f328e2802ae871abe4ca41b4ea79f816R100